### PR TITLE
Mikaels footeradding on all pages

### PR DESCRIPTION
**Pull request by Maatala**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -36,9 +37,44 @@ <h1 class="page-title">>>git config --global user.name team-macedonia</h1>
...
+            </div>
+            <div class="column__footer">
+                <h3>TEAM MACEDONIA</h3>
+                <ul>
+                    <li>Person 1</li>
+                    <li>Person 2</li>
+                    <li>Person 3</li>
+                    <li>Person 4</li>
+                    <li>Person 5</li>
+                </ul>
```
Comment (Alexander-Lindstroem) [inline about.html:54]: You could add our name now!

Comment (PriyeshIllam) [reply about.html:54]: Priyesh Illam

**Review by PriyeshIllam — APPROVED** [review]
LGTM

File: about.html (hunk 1 above)
Comment (idacarlin95) [reply about.html:54]: Caroline Lindbom

Comment (SSSpac) [reply about.html]: Saifelislam Eisa

**Pull request merged**